src/lib.rs

```diff
@@ -94,6 +94,26 @@ impl<T> Node<T> {
             value,
         }
     }
+
+    fn map_value<U>(self, transform: impl FnOnce(T) -> U) -> Node<U> {
```

Comment:
Naming things is hard, but personally I think `map` would suffice as there is not really any chance of mapping anything but the value.

Reply:
Heh, I was gonna add "lemme know if you'd prefer just `map()`".

👍